llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -9328,10 +9328,25 @@ AArch64TargetLowering::LowerCall(CallLoweringInfo &CLI,
       }
       unsigned LocMemOffset = VA.getLocMemOffset();
       int32_t Offset = LocMemOffset + BEAlign;
-      SDValue PtrOff = DAG.getIntPtrConstant(Offset, DL);
-      PtrOff = DAG.getNode(ISD::ADD, DL, PtrVT, StackPtr, PtrOff);
```

Comment:
Was this dead code?

Reply:
Yes, originated here: https://github.com/llvm/llvm-project/commit/09cc56431078f38ad2b992b50f316a51da643cb0.